### ISSUE TITLE
Using a mapped parameter (league, server, race) after ~bnet and ~player yields results for name = *

### ISSUE THREAD

**Issue by shortland** [shortland/Discord-Bot-Starcraft2-Ranked-Stats#2]
`~player diamond`

(league, server, race)

Yields the creation of the below query.

```
'SELECT COUNT(*) FROM `everyone` WHERE `name` LIKE "%%"  AND `league` = "diamond" ';
```

```
'SELECT * FROM `everyone` WHERE `name` LIKE "%%"  AND `league` = "diamond" ORDER BY `mmr` DESC LIMIT 0, 15';
```

**Comment by shortland** [comment]
`~player terran` queries all players that are terran.

`~player Terran` queries all players with the name 'Terran'.

This isn't supposed to be case-sensitive.

**Comment by shortland** [comment]
Implemented fixes in [9058397](https://github.com/shortland/TaeJa-Bot/commit/9058397dc08a235f6fa851e56aab8d2a350a4a79) and [3e46f66](https://github.com/shortland/TaeJa-Bot/commit/3e46f6671778018067c5e46c1e04f7ec5a2e1fcf)
